contrast_security/user_api/application_api.py

```diff
@@ -128,11 +128,11 @@ def filter_applications(self, org_uuid, application_filter=None):
         if application_filter is None:
             application_filter = ApplicationFilter()
         path = '{org_uuid}/applications/filter'.format(org_uuid=org_uuid)
-        return self._post(path, params=application_filter.get_params_as_json())
+        return self._get(path, params=application_filter.get_params_as_json())
```

Comment:
GET endpoint is deprecated since 3.7.10
https://github.com/Contrast-Security-Inc/teamserver/blob/96f9b78e0d020b003c8931bb8b0c109a8833c8b5/teamserver-app/src/main/java/contrast/teamserver/rest/ng/application/NgApplicationsRestController.java#L397

we need to use POST version of this endpoint

Reply:
@cgperez This doesn't work with post and causes an error with teamserver because the body and parameters are not built properly. I wanted to get these out to fix these methods and created this INT-1092 to update these methods to properly use the new POST versions of the endpoints. Moreover, INT-1094 is to update the old methods to use the new apis in general.